addons/mail/static/src/core/web/recipients_input.js

```diff
@@ -84,10 +84,10 @@ export class RecipientsInput extends Component {
                         [
                             ["id", "not in", Array.from(partnerIds)],
                             "|",
-                            ["name", "ilike", name],
+                            ["complete_name", "ilike", name],
```

Comment:
shouldn't it be display_name instead?
complete_name seems to allow searching for the name of the parent record, but it doesn't allow searching for the specific type part, such as "Invoice".

The issue is there for both the quick search and the modal, "Invoice" line doesn't show.

If we need a working example, when searching to add a follower with "Invoice", it does show as expected.